pkg/providers/oidc/verifier.go

```diff
@@ -53,13 +54,25 @@ func (v *idTokenVerifier) Verify(ctx context.Context, rawIDToken string) (*oidc.
 		return nil, fmt.Errorf("failed to parse default id_token claims: %v", err)
 	}
 
-	if isValidAudience, err := v.verifyAudience(token, claims); !isValidAudience {
+	if hasVerifiedClaims, err := v.verifyClaims(token, claims); !hasVerifiedClaims {
 		return nil, err
 	}
 
 	return token, err
 }
 
+func (v *idTokenVerifier) verifyClaims(token *oidc.IDToken, claims map[string]interface{}) (bool, error) {
+	if isValidAudience, err := v.verifyAudience(token, claims); !isValidAudience {
+		return false, err
```

Comment:
Errors should be wrapped with context

Reply:
I am quite new to Go. How can I wrap the errors with context? 
Change Line 66 into: `return false, fmt.Errorf("verifyAudience: %w", err)`?
Change Line 70 into: `return false, fmt.Errorf("verifyEmail: %w", err)`?